### ISSUE TITLE
[Accessibility]: Multiple `H1`s on Batch Changes list page

### ISSUE THREAD

**Issue by courier-new** [sourcegraph/sourcegraph#38783]
### Audit type

ARC Toolkit

### Problem description

On the batch changes list page (`/batch-changes`), there are multiple `<H1/>` elements: the Sourcegraph logo in the navigation bar, and the "Batch Changes" page title, which produces a warning in ARC Toolkit:

<img width="1945" alt="image" src="https://user-images.githubusercontent.com/8942601/178868153-608d495a-345a-4769-8eea-0a91d18af130.png">

### Expected behavior

- The "Batch Changes" page title should be an `<H2/>`, but retain the current style.
- All other `<H2/>`s on the page should be downscaled to `<H3/>`, but retain their current styles.
- No warnings are reported under ARC Toolkit's "Structure and Semantics" tests.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.
